src/components/Sidebar.jsx

```diff
@@ -318,7 +308,7 @@ const Sidebar = ({
                 )
               )
             })}
-            {!IS_DEVICE_IOS ? <NotificationToggle /> : null}
+            {/* {!IS_DEVICE_IOS ? <NotificationToggle /> : null} */}
```

Comment:
do we need this LOL

Reply:
so leave it as comment? can we just remove it?

Reply:
i kept it in because most of the logic is built already, so if we decide to use it in the future then we won't have to rebuild it from scratch